run_experiment.py

```diff
@@ -122,19 +131,39 @@ def parse_arguments():
     dir_opts.add_argument('-o', '--output_file_name', help='The full path name to use when saving the plot.')
 
     args = parser.parse_args()
+
+    if args.log_dir is None:
+        args.log_dir = args.model_dir + 'losses.log'
+
     return args
 
 
 def main(args):
     """ The primary entry point for carrying out experiments on pretrained models.
     """ 
-    experiment_name = args.experiment
+    formatter = logging.Formatter('%(asctime)s %(levelname)s - %(funcName)s: %(message)s',
```

Comment:
that's exactly the sme as in main.py. Don't duplicate code, make a function that retruns that and call it in both files (the function should take `name=__name__` as argument though)